packages/hooks/src/useRegisterPasskey.ts

```diff
@@ -25,22 +25,18 @@ const useRegisterPasskey = () => {
     // the errors are connected with terminating the registration process or setting up the unlock method from user side
     const excluded_error_names = ['NotAllowedError', 'AbortError', 'NotReadableError', 'UnknownError'];
 
-    const [is_passkey_registration_started, setIsPasskeyRegistrationStarted] = React.useState(false);
     const [is_passkey_registered, setIsPasskeyRegistered] = React.useState(false);
     const [passkey_registration_error, setPasskeyRegistrationError] = React.useState<TError | null>(null);
     const [public_key, setPublicKey] = React.useState<null | PublicKeyCredentialCreationOptionsJSON>(null);
 
     const clearPasskeyRegistrationError = () => setPasskeyRegistrationError(null);
-    const cancelPasskeyRegistration = () => setIsPasskeyRegistrationStarted(false);
```

Comment:
why are we removing this?

Reply:
due to refactored code, we don't need this functionality